engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/NewGameScreen.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.nui.layers.mainMenu;
+
+import org.terasology.assets.ResourceUrn;
+import org.terasology.rendering.nui.CoreScreenLayer;
+import org.terasology.rendering.nui.WidgetUtil;
+
+public class NewGameScreen extends CoreScreenLayer {
+
+    public static final ResourceUrn ASSET_URI = new ResourceUrn("engine:newGameScreen");
+
+    @Override
+    public void initialise(){
+
+        AdvancedGameSetupScreen advancedSetupGameScreen = getManager().createScreen(AdvancedGameSetupScreen.ASSET_URI, AdvancedGameSetupScreen.class);
+        WidgetUtil.trySubscribe(this, "advancedSetup", button ->
+            triggerForwardAnimation(advancedSetupGameScreen)
+        );
+
+        WidgetUtil.trySubscribe(this, "close", button ->
+                triggerBackAnimation()
+        );
+    }
```

Comment:
No "play" button :slightly_frowning_face:

Reply:
Nowhere for the play button to go right now 🙂